### ISSUE TITLE
Email Providers: Remove redundant skip button

### ISSUE THREAD

**Issue by taipeicoder** [Automattic/wp-calypso#95923]
### Quick summary

There are two skip buttons on screen. Both take users to the checkout page. Consider removing the top right one since it's not a navigation pattern we see much on dotcom.

![Image](https://github.com/user-attachments/assets/d9c058b4-dd84-423e-8332-73e8469baad3)

### Steps to reproduce

1. Start at /domains/manage/:domain
2. Click on "Add new domain"
3. Select any domain
4. On the email providers screen, there will be two skip buttons

### What you expected to happen

To see only one skip button.

### What actually happened

Saw two skip buttons.

### Impact

One

### Available workarounds?

There is no user impact

**Comment by Robertght** [comment]
I can confirm this is still a thing.

Keeping this report and flagging it to @Automattic/yolo

**Comment by StevenDufresne** [comment]
## Option: Remove the skip button from the cards
Removing the Skip button from the email cards themselves is a relatively easy fix (remove the use of [getUpsellProps](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/email/email-providers-comparison/stacked/provider-cards/get-upsell-props.ts)), but I am personally not comfortable making that change for two reasons:

1. If you choose Google Workspace, the top skip is out of view.
2. In order to progress to the cart, you would need to choose "Purchase" which wouldn't be intuitive because they are within the context of the cards.

## Option: Remove the skip link on the top

The top "Skip" link is part of other purchase flows and while easier to remove, probably somewhat expected by users as the skipping mechanism.

### Example 
<img width="300" alt="Image" src="https://github.com/user-attachments/assets/0cceecae-e719-4d68-9e52-37609c1f5783" />

## Thoughts
In order to understand which button to remove, we'd need a really good sense of what impact either may have, as such, I don't know that it's worth making this change at this time. I think we need to take a more serious look at how we move users through this flow first.

What do you think?

**Comment by taipeicoder** [comment]
@StevenDufresne I agree that this should be addressed on a flow-level rather than looking at the screen in isolation. Details such as the use of arrows instead of chevrons is not consistent with the latest designs.
